homatransport/src/transport/VectioTransport.cc

```diff
@@ -345,8 +357,8 @@ VectioTransport::processDataPkt(HomaPkt* rxPkt)
 {
     if(logEvents){
         logFile << simTime() << " Received data pkt for msg: " << 
-        rxPkt->getMsgId() << " at the receiver: " << 
-        rxPkt->getDestAddr() << std::endl;
+        rxPkt->getMsgId() << " at the receiver: " << rxPkt->getDestAddr() << 
+        " size: " << rxPkt->getDataBytes() << std::endl;
```

Comment:
rxPkt->getMsgId() << " at the receiver: " << rxPkt->getDestAddr() 
                                <<  " size: " << rxPkt->getDataBytes() << std::endl;